pkg/auth/dependency/audit/password_checker.go

```diff
@@ -332,6 +343,38 @@ func (pc *PasswordChecker) ValidatePassword(payload ValidatePasswordPayload) err
 	return PasswordPolicyViolated.NewWithCauses("password policy violated", violations)
 }
 
+// PasswordPolicy outputs a list of PasswordViolation to reflect the password policy.
+func (pc *PasswordChecker) PasswordPolicy() (out []PasswordViolation) {
```

Comment:
It seems now `PasswordViolation` means a specific configured password policy, instead of an instance of violation information. I think it's better to change its name to reflect the current role, e.g. `PasswordPolicy`?